android/src/main/java/com/thebluealliance/androidclient/database/Database.java

```diff
@@ -128,7 +128,6 @@
             + DistrictsTable.KEY + " TEXT PRIMARY KEY NOT NULL, "
             + DistrictsTable.ABBREV + " TEXT NOT NULL, "
             + DistrictsTable.YEAR + " INTEGER NOT NULL, "
-            + DistrictsTable.ENUM + " INTEGER NOT NULL,"
```

Comment:
Any compatibility problems with existing tables in already-installed apps?

Reply:
Shouldn't be - this just leaves the data hanging (not a huge deal since it's pretty small)